ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/tunnel_interfaces.py

```diff
@@ -43,7 +43,7 @@ def tunnel_interfaces(self) -> list | None:
                     }
 
                     if internet_exit_policy["type"] == "zscaler":
-                        tunnel_interface["nat_profile"] = "VRF-AWARE-NAT"
+                        tunnel_interface["nat_profile"] = self.get_internet_exit_nat(internet_exit_policy["type"])[1]["name"]
```

Comment:
Let's create a helper function to just create the name, and then call that from both places instead. Thanks

Reply:
or add it in the internal data model we create.